tap_asana/streams/stories.py

```diff
@@ -76,9 +80,19 @@ def get_objects(self):
             for project in self.call_api("projects", workspace=workspace["gid"]):
                 project_ids.append(project["gid"])
 
+        projects_total = len(project_ids)
+        projects_fraction = projects_total // 100 # near 1% of total projects
+
         # iterate over all project ids and continue fetching
-        for project_id in project_ids:
-            for task in self.call_api("tasks", project=project_id):
+        LOGGER.info("Fetching stories...")
+        for indx, project_id in enumerate(project_ids, 1):
+            if (indx % projects_fraction == 0):
+                LOGGER.info(f"Fetching done for projects: {indx - 1}/{projects_total}")
+            for task in self.call_api(
+                "tasks",
+                project=project_id,
+                modified_since=modified_since,
+                ):
```

Comment:
adding extra query_params `modified_since` as we don't need to make API requests to get stories for tasks that won't be pulled